go.mod

```diff
@@ -44,7 +44,6 @@ require (
 	github.com/gliderlabs/ssh v0.1.1 // indirect
 	github.com/go-ole/go-ole v1.2.1 // indirect
 	github.com/go-openapi/jsonreference v0.17.0
-	github.com/go-openapi/loads v0.19.0 // indirect
```

Comment:
I think some of these are added by the code generation. They are indirect dependencies. Probably will break the lint which atm is checking for differences in these files.

cc @hferentschik

Reply:
Yeah I think this is why this fails, thanks :)